### PR TITLE
READMEs

### PR DESCRIPTION
**Pull request by markstur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,332 @@
...
+
+Whether you choose DSE or an open source distribution of Cassandra, DataStax is there to provide support, consulting and much more. Even if you start with the open source version of Cassandra, you won't get far before you realize that DataStax is helping you. For example, in this code pattern we'll need a client driver to connect to Cassandra from Node.js. That driver is provided by DataStax. We also use the CQLSH tool to interact with the database. We download that tool from DataStax.
+
+## NoSQL, SQL, and CQL
+
+Cassandra is often considered a "NoSQL" database, but as you use it, you will see a language that sure looks like SQL. That's awkward. First of all, it might help to say that "NoSQL" is short for "not only SQL" (or maybe just don't use that term at all). The key thing to remember is that Cassandra was designed to excel and some things that traditional relation databases are not so good at. Along with the benefits of this specialization, Cassandra also comes with some things that you cannot or should not do.
```
Comment (rhagarty) [inline README.md:31]: "excel and some things"

File: doc/source/crc.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+Create your project using the `oc` command line.
+
+```bash
+oc new-project <project-name>
+```
+
+## Configure webhooks
+
+Edit the config YAML in the `config` directory of your cloned repo:
+
```
Comment (rhagarty) [inline doc/source/crc.md:14]: no instructs for cloning repo?

File: doc/source/crc.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+```
+
+### Create persistent volumes
+
+1. Edit the yaml in config/storage.yaml to specify your master node from above
+1. Use `oc create` to create persistent volumes:
+
+```bash
+cd config
+vi storage.yaml)
```
Comment (rhagarty) [inline doc/source/crc.md:62]: paren?

**Review by rhagarty — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (markstur) [reply README.md:31]: fixed

File: doc/source/crc.md (hunk 2 above)
Comment (markstur) [reply doc/source/crc.md:14]: I'll add a TODO in the crc.md.   I'd like to commit these notes but then I need to fix or hide it ASAP.  Will do a separate PR for the config files.

File: doc/source/crc.md (hunk 3 above)
Comment (markstur) [reply doc/source/crc.md:62]: fixed

**Review by rhagarty — APPROVED** [review]

**Pull request merged**